src/Strong/Numeric/Conversions.cs

```diff
@@ -14,112 +16,63 @@ public static class Conversions
         public static int FromWrittenNumberToInt(this string str)
         {
             str = NormalizeWrittenNumber(str);
-            var multiplier = str.StartsWith("negative") ? -1 : 1;
-            str = str.RemoveFirstOcurrence("negative");
-
-            var numberBuilder = new StringBuilder();
+            var isNegative = str.StartsWith(NEGATIVE_KEYWORD);
+            str = str.RemoveFirstOcurrence(NEGATIVE_KEYWORD);
 
-            numberBuilder.Append(CreateSlice("trillion", ref str));
-            numberBuilder.Append(CreateSlice("billion", ref str));
-            numberBuilder.Append(CreateSlice("million", ref str));
-            numberBuilder.Append(CreateSlice("thousand", ref str));
-            numberBuilder.Append(CreateSlice(null, ref str));
-
-            return int.Parse(numberBuilder.ToString()) * multiplier;
-        }
+            var numberBuilder = 0;

```

Comment:
The name `numberBuilder` is weird on this new implementation. Rename to something like `accumulator`.

Noice implementation. Could u send me ur CV?

Reply:
Agree about the naming.
Just sent it on your email 👀